InnovationChallengeLoginPage/InnovationChallengeLoginPage/ContentView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ContentView.swift
+//  InnovationChallengeLoginPage
+//
+//  Created by Aarav Wattal on 2/21/21.
+//
+
+// comment
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        Text("Hello, world!")
+            .padding()
+    }
+}
+
+struct ContentView_Previews: PreviewProvider {
+    static var previews: some View {
+        ContentView()
+    }
+}
```

Comment:
I think this file is not needed - right? It's a leftover from back before the project was renamed.